app/views/single_page_subscriptions/show.html.erb

```diff
@@ -45,6 +45,11 @@ end %>
 
 <%= form_tag single_page_new_session_path, method: :post, :"data-module" => "explicit-cross-domain-links" do %>
 
+  <%= render "govuk_publishing_components/components/inset_text", {
+  } do
```

Comment:
Just curious are the empty `{ }` needed with this syntax?

Reply:
lemme check, I'll admit this was a copy paste from futher up the doc

Reply:
it still works! Have amended this commit and added another to remove from the other.

Still renders:
![image](https://user-images.githubusercontent.com/3694062/158189685-77401669-e04a-4e91-90ee-3a072538a269.png)